test_site/pages/home_with_excluded_elements.rb

```diff
@@ -0,0 +1,16 @@
+class TestHomePageWithExcludedElements < SitePrism::Page
+  set_url('/home_with_excluded_elements.htm')
+  set_url_matcher(/home_with_excluded_elements\.htm$/)
+
+  # individual elements
+  element(:welcome_header, :xpath, '//h1')
+  element(:welcome_message, :xpath, '//span')
+  element(:go_button, :xpath, '//input')
```

Comment:
Similar to above, I don't think most of this is used, right?  
2 or 3 elements is probably sufficient.